src/connectivity/eligibility/search/connectivity-eligibility-search.v6.service.js

```diff
@@ -0,0 +1,23 @@
+angular.module("ovh-api-services").service("OvhApiConnectivityEligibilitySearchV6", function ($resource) {
+    "use strict";
+
+    return $resource("/connectivity/eligibility/search", {
+    }, {
+        getCities: {
```

Comment:
`getCities` and `getStreets` are both declared and are using a `POST` method. It sounds a bit strange to prefix them with `get`. Isn't it?

Reply:
The misunderstanding is related to the fact that these search routes are served as POST methods, where it should be served as GET.

I'll change to `searchCities` / `searchStreets` to avoid this.

Reply:
@JeremyDec Thanks!

Just to inform you that the `dist` folder did not be updated after changes.

It means that your commit is not fully functional.

Please do not use `c4f028e` for any staging purpose. I will create a new release to fit your needs.

Hope it helps,
Thanks!